rust_src/src/remacs_sys.rs

```diff
@@ -89,6 +89,7 @@ extern "C" {
         depth: EmacsInt,
         sexpflag: bool,
     ) -> LispObject;
+    pub fn char_width (c: libc::c_int, dp: *mut Lisp_Char_Table) -> ptrdiff_t;
```

Comment:
Use the declaration from bindings.rs, since it is automatically generated and will thus keep up with any changes to the C function. It'll probably never change until we remove it, but that's a small amount of future labor saved.